hr_applicant/__init__.py

```diff
@@ -1,22 +1,5 @@
 # -*- coding: utf-8 -*-
-##############################################################################
-#    OpenERP, Open Source Management Solution
-#    Copyright (C) 2011 Serpent Consulting Services (<http://serpentcs.com>).
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-#############################################################################
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
```

Comment:
not part of odoo.

Reply:
I will update the license terms in the files.